### DISCUSSION TITLE
Broadlink e password wifi lunghe

### DISCUSSION THREAD

**Discussion by reiner348** [indomus/forum#1697] [Hardware & Firmware]
Ciao a tutti! Vorrei integrare i miei split del condizionatore dentro home assistant, però sono "poco intelligenti" e quindi dovrei passare per smartIR. 
Nella guida vedo che fa riferimento sempre ai broadlink come attuatori, però ho già avuto brutte esperienze quando ancora usavo le app tipo smartlife e quella di broadlink perchè non c'era modo do collegare il dispositivo alla mia wifi... Io ho una password da 32 caratteri, il broadlink l'accetta solo da 30.
L'ultima volta ho provato a marzo 2021, per caso ci sono stati aggiornamenti ai broadlink per superare questo problema da dispositivi preistorici?

**Comment by indomus** [comment]
Buona domanda, sarebbe pure ora. Immaginiamo di sì, ma a dirtela tutta non abbiamo effettuato prove specifiche di recente.

**Comment by reiner348** [comment]
Eh... Prima di comprarne un altro col rischio di rispedirlo indietro di nuovo volevo essere sicuro di poterlo usare... Qualcuno che ne ha uno e si offre volontario per una prova? 😃

**Reply by reiner348** [reply]
Aggiornamento: ho rischiato e comprato un altro broadlink R4 Mini, sperando in un update, perchè ho letto un sito (non so se posso mettere il link) dove spiegano un sistema tramite python per dare in pasto i dati della wifi al dispositivo senza passare per l'app.
Appena arrivato, prima ho provato comunque a scaricare l'app broadlink e configurare la wifi tramite quella, ed ha funzionato senza problemi! Devono aver aggiornato qualcosa e ora il limite sulla lunghezza password wifi è di almeno 34 caratteri (la mia). Quindi in sostanza ho seguito alla lettera la guida indomus!
Ora ho il condizionatore della camera della bambina sotto controllo Home Assistant e Alexa! 👍

**Reply by KC-inDomus** [reply]
grande :) buona info per tutti.
